pkg/porter/publish_test.go

```diff
@@ -0,0 +1,57 @@
+package porter
+
+import (
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestPublish_PorterYamlExists(t *testing.T) {
+
+	p := NewTestPorter(t)
+	p.TestConfig.SetupPorterHome()
+	pwd, err := os.Getwd()
+	require.NoError(t, err, "should not have gotten an error")
+	t.Log(p.TestConfig.TestContext.FindBinDir())
+	p.TestConfig.TestContext.AddTestDirectory("testdata", pwd)
+	opts := PublishOptions{}
+	err = opts.Validate(p.Porter)
+	require.NoError(t, err, "should have no error")
+}
+
+func TestPublish_PorterYamlDoesNotExist(t *testing.T) {
+	p := NewTestPorter(t)
+	p.TestConfig.SetupPorterHome()
+	opts := PublishOptions{}
+	err := opts.Validate(p.Porter)
+	require.Error(t, err, "should have no error")
```

Comment:
the error message here is confusing, can you remove it or fix it? We actually do want an error.